packages/docs/docs/self-hosting/considerations.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 1
+---
+
+# Self-hosting vs Cloud
+
+Self-hosting Medplum can be the right choice for certain use cases, but it comes with significant operational overhead that many organizations underestimate. This guide helps you evaluate whether self-hosting aligns with your needs and resources.
+
+## When Self-hosting Makes Sense
+
+Self-hosting may be appropriate if you have one of these compelling reasons:
+
+**Compliance Requirements**
+You need data sovereignty or have contractual obligations that require on-premises deployment, such as EU regulations or US federal government work. 
+
+**On-Device or Small On-Prem FHIR Server**
+You might need to run a FHIR server on a medical device you are developing, or on a small machine for a rural clinic, and a lightweight Medplum instance with minimal interactions is sufficient. 
+
+**Learning Project**
+You want to gain experience with self-hosting infrastructure, and your Medplum instance will only be used for personal projects without scale requirements.
+
+**Fork Requirements**
+You have a [highly compelling technical reason](/blog/so-youre-thinking-about-forking) to fork the Medplum codebase that cannot be addressed through configuration or our extension points.
+
+## Hidden Costs of Self-hosting
+
+Before choosing self-hosting, carefully consider these often-overlooked operational costs:
+
+**Ongoing Maintenance Burden**
+From our experience, you'll need at least 0.5 FTE (full-time equivalent) dedicated to maintaining your Medplum instance during normal quarters. This includes general maintenance for system stability and ensuring you stay current with version upgrades—an area where many self-hosting customers struggle due to competing priorities and loss of organizational knowledge. 
+
+During quarters requiring *large-scale maintenance projects* like PostgreSQL version upgrades or major Medplum version upgrades, resource requirements can scale to 3+ FTEs.
```

Comment:
This can vary a ton depending on the organization and the workload.  3+ FTE would be the high end.  We don't want to be alarmist with these estimates.